HostLibraryTests/ContractionLibraryLoading_test.cpp

```diff
@@ -52,8 +52,6 @@ TEST(ContractionLibraryLoadingTest, MultipleKernels)
         EXPECT_EQ(solution->name(), "Cijk_Ailk_Bljk_SB_MT128x128x08_K1");
     }
 
-    return;
```

Comment:
cppcheck: Statements following return, break, continue, goto or throw will never be executed.

Reply:
Tests still pass after removing this